common/src/main/java/com/wynntils/overlays/minimap/MinimapOverlay.java

```diff
@@ -178,24 +182,24 @@ public void render(PoseStack poseStack, MultiBufferSource bufferSource, float pa
                     textureZ,
                     width * extraFactor,
                     height * extraFactor,
-                    zoomRenderScale);
+                    1f / zoomRenderScale);
```

Comment:
Turns out this was ported wrongly when we added the new map zooming. For some reason, noone noticed.

Reply:
What was the effect of this bug?

Reply:
The zoom levels were off, in a subtle way. It had given me trouble, as I've unified feature location calculations, including rotation, scaling, and zoom levels. I've had code that worked perfectly on the maps, but not on the minimap. I suspected something was off in this port, but I found the different values (1/scale vs scale) for the main and mini maps. From testing and reading the old code, I'm confident that this is correct.